src/tools/devtools/release-notes/release-notes-2.10.0-src.md

```diff
@@ -1,23 +1,42 @@
 # DevTools 2.10.0 release notes
 
-Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
+The 2.10.0 release of the Dart and Flutter DevTools
+includes the following changes among other general improvements.
+To learn more about DevTools, check out the
+[DevTools overview](https://docs.flutter.dev/tools/devtools/overview).
 
```

Comment:
**question:** Does DevTools pull this release note as a single Markdown file or a rendered HTML page? According to [this include](https://github.com/flutter/website/pull/9325/files#diff-34fa2c96c654719ae827c942cb1ea0533d21242f931522b114661b0bf4f05d20R7), it must not.

I ask because this would be better served as an include with a variable to ensure consistency.

Reply:
At least currently we can't use constructors like the Jekyll variables or Liquid includes. I'll consider ways to do so in the future.